.isort.cfg

```diff
@@ -1,5 +0,0 @@
-[settings]
```

Comment:
This should have been deleted previously. We now have the settings in `pyproject.toml`.